src/charactersheet/viewmodels/dm/encounter_sections/treasure_section/index.js

```diff
@@ -251,7 +252,7 @@ export function TreasureSectionViewModel(params) {
             keys = DataRepository.armors ? Object.keys(DataRepository.armors) : [];
         } else if (self.itemType() == 'item') {
             keys = DataRepository.items ? Object.keys(DataRepository.items) : [];
-        } else if (self.itemType() == 'magicItem') {
+        } else if (self.itemType() == 'magic_item') {
```

Comment:
Can you convert these string constants to an all caps constant? 

- `treasures.MAGIC_ITEM`
- `treasures.ITEM`
etc...